### PR TITLE
Topic tuneup - payment folder, pt 1 - GH 831

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is part 1 of the clean up effort for #831 and addresses the topics in the payment folder in 2.3.x and can be forward ported to 2.4.x. Upon walkthrough of the procedures, some screens were also added or updated as needed and for VBE payments (Amazon Pay and Klarna), removal of screens and information in favor of links to the vendor docs. Also reorganization for PayPal to streamline and simplify the excessive nesting and changes in config UI. Other basic tuneup items: 

- Captions using sentence case
- Headings using sentence case and not infinitives
- Consistency in numbered step structure and style
- Edits for clarity

Pt 2 will address deprecated payment types that will not be ported forward.

Internal staging build at /16/user-guide/v2.3/

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/v2.3/payment/amazon-pay-customer-experience.html
https://docs.magento.com/user-guide/v2.3/payment/amazon-pay-manage.html
https://docs.magento.com/user-guide/v2.3/payment/amazon-pay-resources.html (removed, combined with main page)
https://docs.magento.com/user-guide/v2.3/payment/amazon-pay-seller-central.html (removed, combined with setup page)
https://docs.magento.com/user-guide/v2.3/payment/amazon-pay-setup.html
https://docs.magento.com/user-guide/v2.3/payment/amazon-pay.html
https://docs.magento.com/user-guide/v2.3/payment/bank-transfer.html
https://docs.magento.com/user-guide/v2.3/payment/cash-on-delivery.html
https://docs.magento.com/user-guide/v2.3/payment/check-money-order.html
https://docs.magento.com/user-guide/v2.3/payment/payment-methods.html
https://docs.magento.com/user-guide/v2.3/payment/payment-on-account.html
https://docs.magento.com/user-guide/v2.3/payment/purchase-order.html
https://docs.magento.com/user-guide/v2.3/payment/zero-subtotal-checkout.html
https://docs.magento.com/user-guide/v2.3/payment/merchant-location.html
https://docs.magento.com/user-guide/v2.3/payment/online-payment-methods.html
https://docs.magento.com/user-guide/v2.3/payment/payments.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-billing-agreements.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-business-account.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-countries.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-credit.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-express-checkout.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-fraud-management-filter.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-in-context-checkout.html (removed, combine with Express Checkout page)
https://docs.magento.com/user-guide/v2.3/payment/paypal-payflow-link.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-payflow-pro.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-payment-solutions.html (removed, combined with main page)
https://docs.magento.com/user-guide/v2.3/payment/paypal-payments-advanced.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-payments-pro.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-payments-standard.html
https://docs.magento.com/user-guide/v2.3/payment/paypal-settlement-reports.html
https://docs.magento.com/user-guide/v2.3/payment/paypal.html
https://docs.magento.com/user-guide/v2.3/payment/klarna-customer-experience.html
https://docs.magento.com/user-guide/v2.3/payment/klarna-manage.html (removed, this was all Klarna UI and info)
https://docs.magento.com/user-guide/v2.3/payment/klarna-setup.html

### CODE REVIEW

File: src/payment/amazon-pay-setup.md [hunk 1]
```diff
@@ -1,118 +1,128 @@
...
+- [Amazon Pay seller credentials][5]
 
 ## Connect to Amazon Pay
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the upper-right corner, set the **Scope** to establish where the Amazon Pay configuration applies.
+1. In the upper-left corner, set the **Scope** to establish where the Amazon Pay configuration applies.
 
 1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/amazon-pay-setup.md:22]:
```suggestion
1. In the left panel, expand ![Expansion selector]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

File: src/payment/amazon-pay-setup.md [hunk 2]
```diff
@@ -1,118 +1,128 @@
...
+   | Charge on Order | Amazon creates the invoice and charges the customer when the order is placed. |
 
-1. Set **Authorization Mode** to one of the following to determine the method that is used to transmit authorization data.
+1. Set **Authorization Mode** to determine the method that is used to transmit authorization data.
 
-   | `Immediate` | (Default) Makes the disposition of an authorization available within a couple of seconds. |
-   | `Automatic` | (Recommended) Uses synchronous authorization if possible, and switches to asynchronous processing as needed, for some declines. The change in processing modes is transparent to the user. The status of sales order is _Pending_ while the disposition of authorization is outstanding. |
+   | Immediate | (Default) Makes the disposition of an authorization available within a couple of seconds. |
+   | Automatic | (Recommended) Uses synchronous authorization if possible, and switches to asynchronous processing as needed for some declines. The change in processing modes is transparent to the user. The status of sales order is _Pending_ while the disposition of authorization is outstanding. |
```
Comment (hickskenh) [inline src/payment/amazon-pay-setup.md:101]:
```suggestion
   | Automatic | (Recommended) Uses synchronous authorization, if possible, and switches to asynchronous processing as needed for some declines. The change in processing modes is transparent to the user. The status of sales order is _Pending_ while the disposition of authorization is outstanding. |
```

File: src/payment/amazon-pay-setup.md [hunk 3]
```diff
@@ -171,33 +181,31 @@ When you have your account information, complete the following sections.
...
 
 ### Developer Options
 
-1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) **Developer Options**.
+1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Developer Options** section.
 
 1. Make sure that **Logging** is set to `Yes`.
 
-    After the setup is complete and logs are generated, you can return to this location and select a link in **Developer Logs** to download to view Client and IPN logs. If logs are not available or have not been generated, a message displays `No logs are currently available.`
+   After the setup is complete and logs are generated, you can return to this location and select a link in **Developer Logs** to download to view Client and IPN logs. If logs are not available or have not been generated, a message displays `No logs are currently available.`
```
Comment (hickskenh) [inline src/payment/amazon-pay-setup.md:192]:
```suggestion
   After the setup is complete and logs are generated, you can return to this location and select a link in **Developer Logs** to download to view Client and IPN logs. If logs are not available or have not been generated, the `No logs are currently available.` message appears.
```

File: src/payment/amazon-pay.md [hunk 4]
```diff
@@ -1,27 +1,36 @@
 ---
 title: Amazon Pay
+redirect_from:
+  - /payment/amazon-pay-resources.html
 ---
 
-
-Amazon Pay and Login with Amazon offer your customers a secure and convenient way to pay that is easy to set up, and that helps to increase conversions and sales. Customers who log in to your store with their Amazon credentials can speed through checkout using the payment and shipping information that is already stored in their Amazon accounts.
+Amazon Pay and _Login with Amazon_ offer your customers a secure and convenient way to pay that is easy to set up, and that helps to increase conversions and sales. Customers who log in to your store with their Amazon credentials can speed through checkout using the payment and shipping information that is already stored in their Amazon accounts.
```
Comment (hickskenh) [inline src/payment/amazon-pay.md:7]: why is one ital and the other not?

File: src/payment/amazon-pay-setup.md [hunk 5]
```diff
@@ -1,118 +1,128 @@
 ---
 title: Setting Up Amazon Pay
+redirect_from:
+  - /payment/amazon-pay-seller-central.html
 ---
 
-Amazon Pay and Login with Amazon work together to create a great buyer experience.
+Amazon Pay and Login with Amazon work together to create a great buyer experience. For more information, see the [Amazon Pay and Login with Amazon extension for Magento 2][1] documentation provided by Amazon.
```
Comment (hickskenh) [inline src/payment/amazon-pay-setup.md:7]: in another topic, Login with Amazon is ital. review for consistency.

File: src/payment/amazon-pay.md [hunk 6]
```diff
@@ -1,27 +1,36 @@
...
 
 For international merchants with a region in EU or UK, Amazon Pay provides a [multi-currency feature][1]. If you list prices in more than one currency, this feature allows a customer to select their currency and pay the exact amount quoted on the site, without concern for conversion or rates.
 
-## Amazon Pay highlights
+{% include VBE-note.md %}
+
+## Feature highlights
 
-- [Shop Safely][2] - Customer purchases are protected by [Amazon’s A-to-z  Guarantee][2]. Customer password and payment information is never shared.
+- [Shop Safely][2] - Customer purchases are protected by Amazon’s A-to-z Guarantee. Customer password and payment information is never shared.
```
Comment (hickskenh) [inline src/payment/amazon-pay.md:15]:
```suggestion
- [Shop Safely][2] - Customer purchases are protected by Amazon’s A-to-Z Guarantee. Customer password and payment information is never shared.
```

File: src/payment/bank-transfer.md [hunk 7]
```diff
@@ -2,44 +2,50 @@
...
 
-Magento allows you to accept payment that is transferred from a customer’s bank account and deposited into your merchant bank account.
+## Configure bank transfer payments
 
-![Bank Transfer Payment]({% link images/images/config-sales-payment-methods-bank-transfer-payment.png %}){: .zoom}
-_Bank Transfer Payment_
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-## To configure bank transfer payments:
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/bank-transfer.md:11]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

File: src/payment/cash-on-delivery.md [hunk 8]
```diff
@@ -2,40 +2,49 @@
 title: Cash On Delivery
 ---
 
-
 Magento allows you to accept COD payments for purchases. You can accept COD payment from only specific countries, and fine-tune the configuration with minimum and maximum order total limits.
```
Comment (hickskenh) [inline src/payment/cash-on-delivery.md:5]:
```suggestion
Magento allows you to accept COD payments for purchases. You can accept COD payment only from specific countries, and you can fine-tune the configuration with minimum and maximum order total limits.
```

File: src/payment/check-money-order.md [hunk 9]
```diff
@@ -2,40 +2,46 @@
 title: Check / Money Order
 ---
 
+Magento allows you to accept payments by check or money order. The Check / Money Order payment method is enabled for your story by default. You can accept checks and money orders from only specific countries, and fine tune the configuration with minimum and maximum order total limits.
```
Comment (hickskenh) [inline src/payment/check-money-order.md:5]:
```suggestion
Magento allows you to accept payments by check or money order. The Check / Money Order payment method is enabled for your store by default. You can accept checks and money orders only from specific countries, and you can fine tune the configuration with minimum and maximum order total limits.
```

File: src/payment/check-money-order.md [hunk 10]
```diff
@@ -2,40 +2,46 @@
...
 
-Magento allows you to accept payments by check or money order. The Check / Money Order payment method is enabled for your story by default. You can accept checks and money orders from only specific countries, and fine-tune the configuration with minimum and maximum order total limits.
+## Configure payment by check or money order
 
-![Check / Money Order]({% link images/images/config-sales-payment-methods-check-money-order.png %}){: .zoom}
-[_Check / Money Order_]({% link configuration/sales/payment-methods.md %})
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-## To configure payment by check or money order:
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/check-money-order.md:11]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

File: src/payment/klarna-customer-experience.md [hunk 11]
```diff
@@ -1,41 +1,40 @@
...
-1. Do one of the following:
+1. The customer does one of the following:
 
-   - Add the purchase to your Klarna account.
-   - Apply for Klarna credit, and receive an instant decision.
+   - Adds the purchase to their Klarna account
+   - Applies for Klarna credit, and receives an instant decision
 
-1. After the terms of the credit are approved, tap <span class="btn">Place Order</span>.
+1. After the terms of the credit are approved, the customer clicks/taps <span class="btn">Place Order</span>.
```
Comment (hickskenh) [inline src/payment/klarna-customer-experience.md:29]: are we switching from click to "click/tap"?   first time i've seen this.

File: src/payment/klarna-customer-experience.md [hunk 12]
```diff
@@ -1,41 +1,40 @@
...
+1. The customer selects **Pay later**.
 
-   The terms appear below. In this example, no sign-up is needed, and the customer can pay in 14 days.
+   The terms appear below. In the following example, no sign-up is needed and the customer can pay in 14 days.
 
    ![Pay Later]({% link images/images/storefront-cart-payment-method-klarna2b.png %}){: .zoom}
    _Pay Later_
 
-1. To pay later with Klarna according to the terms specified, tap <span class="btn">Place Order</span>.
+1. To pay later with Klarna according to the terms specified, the customer clicks/taps <span class="btn">Place Order</span>.
```
Comment (hickskenh) [inline src/payment/klarna-customer-experience.md:40]: clicks/taps again?

File: src/payment/klarna-setup.md [hunk 13]
```diff
@@ -7,55 +7,57 @@ To offer your customers the benefit of paying with Klarna, you must first sign u
...
+Click <span class="btn">Save Config</span> at any time to save your progress.
 
-## Step 2: Configure Magento
+### Step 1: Begin the Magento configuration
 
-1. In a new browser tab, log in to the Admin of your Magento store.
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. On the Admin sidebar, go to **Stores** > _Settings_ > **Configuration**.
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:49]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %})**Sales** and choose **Payment Methods**.
```

File: src/payment/merchant-location.md [hunk 14]
```diff
@@ -3,17 +3,17 @@ title: Merchant Location
...
 
-![]({% link images/images/config-sales-payment-methods-merchant-location.png %}){: .zoom}
-_[Merchant Location]({% link configuration/sales/payment-methods.md %})_
+## Set the merchant location
 
-## To enter the merchant location:
+1. On the Admin sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. On the Admin sidebar, tap **Stores**. Then under **Settings**, choose **Configuration**.
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/merchant-location.md:12]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

File: src/payment/payment-on-account.md [hunk 15]
```diff
@@ -3,38 +3,42 @@ b2b_only: true
...
+   {:.bs-callout-info}
+   If necessary, first deselect the **Use system value** checkbox to change these settings.
+
+1. To allow payment on account, set **Enabled** to `Yes`.
 
-   - After the Enabled field, clear the **Use system value** checkbox. Then, set **Enabled** to `Yes`.
+1. Enter a **Title** to identify the this payment method during checkout, or you can accept the `Payment on Account` default title.
 
-   - If you want to change the default title, “Payment on Account,” clear the **Use system value** checkbox. Then, enter the title that you want to appear during checkout.
+1. If orders typically wait for approval, accept the default **New Order Status** as `Pending”` until it is approved.
```
Comment (hickskenh) [inline src/payment/payment-on-account.md:26]:
```suggestion
1. If orders typically wait for approval, accept the default **New Order Status** as `Pending` until it is approved.
```

File: src/payment/paypal-credit.md [hunk 16]
```diff
@@ -2,19 +2,17 @@
 title: PayPal Credit
 ---
 
-PayPal Credit offers your customers quick access to financing, so they can buy now and pay over time, at no additional cost to you. You are not charged when customers choose PayPal Credit, and pay only your normal PayPal transaction fee.
+PayPal Credit offers your customers quick access to financing, so they can buy now and pay over time, at no additional cost to you. You are not charged when customers choose PayPal Credit, and pay only your normal PayPal transaction fee. To learn more, see [PayPal Credit][1] on the PayPal website.
```
Comment (hickskenh) [inline src/payment/paypal-credit.md:5]:
```suggestion
PayPal Credit offers your customers quick access to financing, so they can buy now and pay over time, at no additional cost to you. You are not charged when customers choose PayPal Credit, and you pay only your normal PayPal transaction fee. To learn more, see [PayPal Credit][1] on the PayPal website.
```

File: src/payment/paypal.md [hunk 17]
```diff
@@ -1,41 +1,57 @@
...
 
-- [Recommended Solutions (Paypal Express Checkout) ]({% link payment/paypal-express-checkout.md %})
-- PayPal All-In-One Payment Solutions
-- PayPal Payment Gateways
+## PayPal business account
 
-The selection of available PayPal solutions varies by merchant location. PayPal Express Checkout and PayPal Payments Standard can be used in all parts of the world. To learn more, see: [PayPal Solutions by Country]({% link payment/paypal-countries.md %}).
+To offer PayPal as a payment method in your store, you must have a PayPal [business account][1] and/or a PayPal Payflow account. The account requirements are specified in the description of each PayPal solution. Your PayPal merchant account is also used to manage any [fraud filters]({% link payment/paypal-fraud-management-filter.md %}) that are applied to purchases made from your store.
```
Comment (hickskenh) [inline src/payment/paypal.md:17]: is there a paypal payflow topic to link to?

File: src/payment/purchase-order.md [hunk 18]
```diff
@@ -6,33 +6,38 @@ A purchase order (PO) allows commercial customers to pay for authorized purchase
...
 
-![Purchase Order]({% link images/images/config-sales-payment-methods-purchase-order.png %}){: .zoom}
-_Purchase Order_
+## Configure payment by purchase order
 
-## To configure payment by purchase order:
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. On the Admin sidebar, tap **Stores**. Then under **Settings**, choose **Configuration**.
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/purchase-order.md:13]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

File: src/payment/zero-subtotal-checkout.md [hunk 19]
```diff
@@ -14,33 +14,37 @@ Zero Subtotal Checkout can be used for orders with a subtotal of zero that are t
...
 
-![Zero Subtotal Checkout]({% link images/images/config-sales-payment-methods-zero-subtotal-checkout.png %}){: .zoom}
-_Zero Subtotal Checkout_
+## Configure zero subtotal checkout
 
-## To configure zero subtotal checkout:
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. On the Admin sidebar, tap **Stores**. Then under **Settings**, choose **Configuration**.
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
```
Comment (hickskenh) [inline src/payment/zero-subtotal-checkout.md:21]:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

**Review by hickskenh — APPROVED** [review]
minor issues marked.  approved after your review.

File: src/payment/amazon-pay-setup.md (hunk 1 above)
Comment (leslietilling) [reply src/payment/amazon-pay-setup.md:22]: The left panel doesn't use that icon, it is only for page section expansion.

File: src/payment/klarna-customer-experience.md (hunk 11 above)
Comment (leslietilling) [reply src/payment/klarna-customer-experience.md:29]: Only when describing storefront functionality, which is more likely to be on a mobile device.

File: src/payment/klarna-customer-experience.md (hunk 12 above)
Comment (leslietilling) [reply src/payment/klarna-customer-experience.md:40]: same ;-)

File: src/payment/paypal.md (hunk 17 above)
Comment (leslietilling) [reply src/payment/paypal.md:17]: I'll look. The PayPal stuff is very convoluted from a product perspective.

File: src/payment/amazon-pay.md (hunk 4 above)
Comment (leslietilling) [reply src/payment/amazon-pay.md]: Was in quotes. I looked at Amazon docs and I think better to not italicize and just do it the way that they do.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**